TS.fsx

```diff
@@ -1086,13 +1086,13 @@ module Emit =
                 "%saddEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;"
                 fPrefix
 
-    let EmitConstructorSignature (i:Browser.Interface) =
+    let EmitConstructorSignature flavor (i:Browser.Interface) =
         let emitConstructorSigFromJson (c: InputJsonType.Root) =
             c.Signatures |> Array.iter (Pt.Printl "%s;")
 
-        let removedCtor = getRemovedItems ItemKind.Constructor Flavor.All  |> Array.tryFind (matchInterface i.Name)
+        let removedCtor = getRemovedItems ItemKind.Constructor flavor  |> Array.tryFind (matchInterface i.Name)
         if Option.isNone removedCtor then
-            let overriddenCtor = getOverriddenItems ItemKind.Constructor Flavor.All  |> Array.tryFind (matchInterface i.Name)
+            let overriddenCtor = getOverriddenItems ItemKind.Constructor flavor  |> Array.tryFind (matchInterface i.Name)
```

Comment:
@mhegazy After looking into this generating script, I found out the reason why `"flavor": "Web"`  in `inputfiles/overridingTypes.json` is ignored is that it always use `Flavor.All` to EmitConstructorSignature.